cli/wallet.go

```diff
@@ -260,6 +260,7 @@ var walletSetDefault = &cli.Command{
 			return err
 		}
 
+		fmt.Println("Default address set to:", addr)
```

Comment:
Don't we want to have a space here?
```suggestion
		fmt.Println("Default address set to: ", addr)
```

Reply:
I originally had a space there, but during testing I got double spaces in the output, which was a bit surprising. This is what I currently get with no space:
```
phi:~$lotus wallet set-default t1lu7zszqwr7emsvjznr7utbsmsjia376focp2gvq
Default address set to: t1lu7zszqwr7emsvjznr7utbsmsjia376focp2gvq
```